apps/web-giddh/src/app/tallysync/inprogress/inprogress.component.scss

```diff
@@ -25,7 +25,7 @@
             box-shadow: none;
             height: 5px;
             margin-bottom: 0px;
-            border: $whisper 1px solid;
```

Comment:
@aditi-tiwari17 @shalinee-walkover Please check this code revert

Reply:
checked, its fine